src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -92,13 +92,16 @@ static int nbd = -1;
 #endif
 #define htonll(a) ntohll(a)
 
-static int do_unmap();
-
```

Comment:
I will remove this declaration, thanks.
I still prefer two commits, since they are different fixes, and will be clearer.